charts/vertical-pod-autoscaler/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- sig-autoscaling-vpa-approvers
+reviewers:
+- sig-autoscaling-vpa-reviewers
```

Comment:
I think we should have 2 new groups:
`sig-autoscaling-vpa-helm-chart-approvers` and `sig-autoscaling-vpa-helm-chart-reviewers`

We can start with the existing approvers/reviewers (or a subset of them, depending if they want in on the helm chart responsibility). 

Then over time we can start adding new contributors to the helm-chart only groups